src/react_native/async_storage.cljs

```diff
@@ -0,0 +1,14 @@
+(ns react-native.async-storage
+  (:require ["@react-native-async-storage/async-storage" :default AsyncStorage]))
+
+
+(defn set-item
+  [key value]
+  (-> (.setItem AsyncStorage (clj->js key) (str value))
```

Comment:
Thank you @ulisesmac for the suggestions. I have noticed that we have another `async-storage` module in `status-im.async-storage.core` which shouldn't be there. I have opened an issue to refactor it to `react-native` namespace and improve the implementation. Also, feel free to share your opinion on the current implementation in `status-im.async-storage.core`.

Reply:
Thanks! I read that  implementation and is looking good since it's using transit as storing format. 
Could you please use that module instead of creating a new one?

If it's not possible to use it (since it's in status-im1), could you please move it in this PR?
I think we should avoid creating repetitive and potential confusing code.

Reply:
Alright, I removed my implementation. I will refactor the namespace in another issue